test/built-ins/Temporal/PlainDateTime/prototype/toZonedDateTime/disambiguation-invalid.js

```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
```

Comment:
Consider naming this test `disambiguation-invalid-string.js` and expanding the invalid strings a bit to match https://github.com/tc39/test262/blob/main/test/built-ins/Temporal/PlainDate/prototype/add/overflow-invalid-string.js (e.g. "earlier" with a dotless i to test Unicode canonicalization, a string with an extra \0 in it)

Reply:
Good idea -- done